vips/vip-491/bsctestnet.ts

```diff
@@ -0,0 +1,179 @@
+export const sUSDe = "0xA3A3e5ecEA56940a4Ae32d0927bfd8821DdA848A";
+export const USDe = "0x8bAe3E12870a002A0D4b6Eb0F0CBf91b29d9806F";
+
+export const VsUSDe_CORE = "0x33e4C9227b8Fca017739419119BbBA33A089D4a0";
+export const VUSDe_CORE = "0x36e8955c305aa48A99e4c953C9883989a7364a42";
+
+export const sUSDe_INITIAL_SUPPLY = parseUnits("10000", 18);
+export const USDe_INITIAL_SUPPLY = parseUnits("10000", 18);
+
+export const convertAmountToVTokens = (amount: BigNumber, exchangeRate: BigNumber) => {
+  const EXP_SCALE = parseUnits("1", 18);
+  return amount.mul(EXP_SCALE).div(exchangeRate);
+};
+
+const vip491 = () => {
+  const meta = {
+    version: "v2",
+    title: "[sepolia] New sUSDe and USDe market in the Core pool",
+    description: ``,
+    forDescription: "I agree that Venus Protocol should proceed with this proposal",
+    againstDescription: "I do not think that Venus Protocol should proceed with this proposal",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds or not",
+  };
+
+  return makeProposal(
+    [
+      // Add sUSDe Market
+      // oracle config
+      {
+        target: sepolia.REDSTONE_ORACLE,
+        signature: "setDirectPrice(address,uint256)",
+        params: [USDe, parseUnits("1", 18)],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sepolia.CHAINLINK_ORACLE,
+        signature: "setTokenConfig((address,address,uint256))",
+        params: [[USDe, CHAINLINK_USDe_FEED, STALE_PERIOD_26H]],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sepolia.RESILIENT_ORACLE,
+        signature: "setTokenConfig((address,address[3],bool[3]))",
+        params: [
+          [USDe, [sepolia.REDSTONE_ORACLE, sepolia.CHAINLINK_ORACLE, sepolia.CHAINLINK_ORACLE], [true, true, true]],
+        ],
+        dstChainId: LzChainId.sepolia,
+      },
+
+      // Market configurations
+      {
+        target: VsUSDe_CORE,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: ["7200"],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sUSDe,
+        signature: "faucet(uint256)",
+        params: [sUSDe_INITIAL_SUPPLY],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sUSDe,
+        signature: "approve(address,uint256)",
+        params: [sepolia.POOL_REGISTRY, 0],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sUSDe,
+        signature: "approve(address,uint256)",
+        params: [sepolia.POOL_REGISTRY, sUSDe_INITIAL_SUPPLY],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sepolia.POOL_REGISTRY,
+        signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+        params: [
+          [
+            VsUSDe_CORE,
+            parseUnits("0.72", 18), // CF
+            parseUnits("0.75", 18), // LT
+            sUSDe_INITIAL_SUPPLY, // initial supply
+            sepolia.VTREASURY,
```

Comment:
I would send the VTokens to the Normal timelock, to burn some of them (around $100). I would do the same for the USDe market

Reply:
7d6269b7684040d0709fc890bc68def29265e10e